core/modules/l4_checksum.cc

```diff
@@ -36,6 +36,8 @@
 #include "../utils/tcp.h"
 #include "../utils/udp.h"
 
+enum { DEFAULT_GATE = 0, _DROP_GATE };
```

Comment:
`_DROP_GATE` could get easily confused with `DROP_GATE` used by the scheduler to discard packets. Perhaps rename these to `VALID_GATE` (or `DEFAULT_GATE`) and `INVALID_GATE`?